packages/popup/src/index.css

```diff
@@ -1,3 +1,126 @@
+/* stylelint-disable  at-rule-no-unknown */
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@layer base {
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Book.ttf)
+      format('truetype');
+    font-weight: 400;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Book-Italic.ttf)
+      format('truetype');
+    font-style: italic;
+    font-weight: 400;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Medium.ttf)
+      format('truetype');
+    font-weight: 500;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Medium-Italic.ttf)
+      format('truetype');
+    font-style: italic;
+    font-weight: 500;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Bold.ttf)
+      format('truetype');
+    font-weight: 700;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Bold-Italic.ttf)
+      format('truetype');
+    font-style: italic;
+    font-weight: 700;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Black.ttf)
+      format('truetype');
+    font-weight: 800;
+  }
+  @font-face {
+    font-family: 'Circular Std';
+    src: url(https://conflux-static.oss-cn-beijing.aliyuncs.com/fonts/Circular-Std-Black-Italic.ttf)
```

Comment:
better remove the url of these fonts, they are not open source, there might be legal issue

Reply:
the font had been purchased before